Lambdas/AutoFaqs/AutoFaqsHandler.swift

```diff
@@ -14,26 +14,40 @@ import Foundation
 #endif
 
 @main
-struct AutoFaqsHandler: LambdaHandler {
-    typealias Event = APIGatewayV2Request
-    typealias Output = APIGatewayV2Response
+@dynamicMemberLookup
+struct AutoFaqsHandler {
+    struct SharedContext {
+        let awsClient: AWSClient
+    }
```

Comment:
These `SharedContex`es don't get initialized multiple times across multiple lambda invocations. That's the reason.